src/app.js

```diff
@@ -11,6 +12,7 @@ app.use(morgan('dev'));
  * Routes configurations
  */
 app.use('/api/v1/lost', routes);
+app.use('/', userRoutes);
```

Comment:
Check the format of the route above, you would do something like `app.use('/api/v1/auth/',userRoutes)`